### PR TITLE
docs: add packages entry

### PR DESCRIPTION
**Pull request by userquin**
Also fixex some wording and discord link (use https protocol)

### CODE REVIEW

**Comment by userquin** [comment]
I'll replace ✅ with icons...

File: docs/extractors/pug.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 title: Pug extractor
 description: Pug extractor for UnoCSS (@unocss/extractor-pug)
+prev: Attributify JSX | Transformers
+next: Inspector | Other Packages
```
[the remark this answers is not in the document]
Comment (userquin) [reply docs/extractors/pug.md:5]: no, maybe we can add some function to add it

[the remark this answers is not in the document]
Comment (userquin) [reply docs/extractors/pug.md:5]: working on it...

[the remark this answers is not in the document]
Comment (antfu) [reply docs/extractors/pug.md:5]: Then I think we could not have it until some built-in support in VitePress

[the remark this answers is not in the document]
Comment (userquin) [reply docs/extractors/pug.md:5]: ok, removing prev/next entries...

File: docs/guide/packages.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: Packages
+description: "UnoCSS Packages: available packages and what's included and enabled in unocss."
+outline: deep
+---
+
+# Packages
```
Comment (antfu) [inline docs/guide/packages.md:7]: At some point, I think VitePress should have some macros to import content from other MD files.

Comment (userquin) [reply docs/guide/packages.md:7]: https://vitepress.dev/guide/markdown#import-code-snippets

**Pull request merged**